### PR TITLE
Brightroll bidder- BusinessInsider changes

### PR DESCRIPTION
**Pull request by smithaammassamveettil**
Description:

- Updated the list of blocked creative types, attributes and categories for business insider
- Added validation for supported vendors

### CODE REVIEW

File: src/main/java/org/prebid/server/bidder/brightroll/BrightrollBidder.java [hunk 1]
```diff
@@ -121,6 +126,10 @@ private String getAndValidateImpExt(Imp imp) {
             throw new PreBidException("publisher is empty");
         }
 
+        if (!supportedVendors.contains(publisher)) {
```
Comment (DGarbar) [inline src/main/java/org/prebid/server/bidder/brightroll/BrightrollBidder.java:129]: I think, you can use `!publisherIdToOverride.keys().contains(publisher)` and remove supportedVendors collections from constructor. 
Or just create `supportedVendors` from `publisherIdToOverride.keys()`.

Comment (smithaammassamveettil) [reply src/main/java/org/prebid/server/bidder/brightroll/BrightrollBidder.java]: Thanks for reviewing @DGarbar , changed to use vendors from map directly.

**Comment by smithaammassamveettil** [comment]
@DGarbar Have a question on the build issue on the integration test. Can see below error on integration testcase
_[ERROR]   BrightrollTest.openrtb2AuctionShouldRespondWithBidsFromBrightroll:62 ext.responsetimemillis
Expected: cache
     but none found_  

After adding the Invalid publisher check, this error is introduced. But can see the publisher "businessinsider" is being used in the request json for integration testcase, which is a valid publisher in the config as well. Is Integration testcase using a different publisher?

**Comment by DGarbar** [comment]
> @DGarbar Have a question on the build issue on the integration test. Can see below error on integration testcase
> _[ERROR] BrightrollTest.openrtb2AuctionShouldRespondWithBidsFromBrightroll:62 ext.responsetimemillis Expected: cache but none found_
> 
> After adding the Invalid publisher check, this error is introduced. But can see the publisher "businessinsider" is being used in the request json for integration testcase, which is a valid publisher in the config as well. Is Integration testcase using a different publisher?

@smithaammassamveettil You need to update `prebid-server-java\src\test\resources\org\prebid\server\it\openrtb2\brightroll\test-brightroll-bid-request-1.json`
With updated parameters.
```
"battr": [
          1,
          3,
          8,
          9,
          10,
          13,
          14,
          17
        ]
```
```
 "bcat": [
    "IAB7",
    "IAB7-39",
    "IAB7-44",
    "IAB9-30",
    "IAB11",
    "IAB13-2",
    "IAB14-1",
    "IAB15-1",
    "IAB15-5",
    "IAB17-18",
    "IAB18-1",
    "IAB18-2",
    "IAB7-19",
    "IAB19-30",
    "IAB23",
    "IAB25-7",
    "IAB26-1",
    "IAB26-2",
    "IAB26-3",
    "IAB26-4"
  ],
```
Because in this test we use `businessinsider` and this test shows us the real request to bidder.

P.S and please update `makeHttpRequestsShouldNotUpdateEachImpIfExtPublisherIsNotInBidderAccounts` to `makeHttpRequestsShouldOverrideBadvAndBcatWhenPublisherIsInBidderAccounts`
and add `.battr(singletonList(1)` to `Banner` and `Video`

**Comment by smithaammassamveettil** [comment]
Thanks @DGarbar  for letting me know that. I have updated the testcases. Please review when you get a chance.

**Comment by smithaammassamveettil** [comment]
Thanks @DGarbar for approving the PR. Is it good to merge? If so, can you please help in merge as well?

**Comment by DGarbar** [comment]
> Thanks @DGarbar for approving the PR. Is it good to merge? If so, can you please help in merging as well?

 It will be merged in the next release.

**Comment by smithaammassamveettil** [comment]
When is the next release?

**Comment by smithaammassamveettil** [comment]
@DGarbar  Got a revised blockedadvertiser list for Business insider recently. So updated the same in yaml and testcase. So your previous approval got dismissed now. Can you please re-approve the PR? Also please let me know when is the next release and when all these changes will be merged or released?

File: src/main/resources/bidder-config/brightroll.yaml [hunk 2]
```diff
@@ -44,11 +44,95 @@ adapters:
           - thebeverlyhillsmdsolution.com
           - thecrux.com
           - thehornnews.com
+          - ezmob.com
```
Comment (DGarbar) [inline src/main/resources/bidder-config/brightroll.yaml:47]: Sorry for the late response. 
We found that DSP doesn't support more that 50 badv.

Comment (bretg) [reply src/main/resources/bidder-config/brightroll.yaml:47]: Sorry, this is my mistake. The Rubicon exchange has a limit of 50 badv entries. Brightroll does not. This PR is ok.

**Comment by smithaammassamveettil** [comment]
@DGarbar Is this released to production now?

**Comment by DGarbar** [comment]
@smithaammassamveettil  I'll notify you when it will be in production.

**Pull request merged**